packages/fxa-shared/test/metrics/amplitude.js

```diff
@@ -488,21 +488,36 @@ describe('metrics/amplitude:', () => {
       assert.isTrue(result);
     });
 
-    it('success - if event_type is `fxa_pay_setup - *` and language is provided', () => {
+    it('success - if event_type is `fxa_pay_setup - *` and language/product/plan/checkoutType is provided', () => {
       const event = {
         ...minimumEvent,
         event_type: 'fxa_pay_setup - view',
         language: 'en',
+        event_properties: {
+          ...minimumEvent.event_properties,
+          product_id: 'product',
+          plan_id: 'plan',
+          checkout_type: 'without-account',
+          subscribed_plan_ids: '',
```

Comment:
Nit: `subscribed_plan_ids` is now optional.

Reply:
👍